moduleframework/dockerlinter.py

```diff
@@ -236,10 +236,11 @@ def check_helpmd_is_present(self):
                  False if help.md is not specified in Dockerfile
         """
         helpmd_present = False
-        for c in self.docker_dict[COPY]:
-            if "help.md" in c:
-                helpmd_present = True
-        for a in self.docker_dict[ADD]:
-            if "help.md" in a:
-                helpmd_present = True
+        for instruction in [COPY, ADD]:
+            try:
+                helpmd = [help for help in self.docker_dict[instruction] if "help.md" in help]
+                if helpmd:
+                    helpmd_present = True
+            except KeyError:
+                print("Instruction %s is not present in Dockerfile", instruction)
```

Comment:
please use logging

Reply:
This should be changed in https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/common.py respectively https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/common.py#L121